src/hooks/useLogDownloader/index.ts

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from "react";
+import { isProduction } from "utils/environmentVariables";
+import { leaveBreadcrumb } from "utils/errorReporting";
+/*
+ * useLogDownloader is a custom hook that downloads a log file from a given URL.
```

Comment:
nit. for consistency
```suggestion
 * `useLogDownloader` is a custom hook that downloads a log file from a given URL.
```